network-store-server/src/main/resources/db/changelog/changesets/move_regulating_to_regulationPoint_20241204T110000Z.sql

```diff
@@ -0,0 +1,29 @@
+-- generator
+update regulatingpoint
+set regulating = (select generator.voltageregulatoron from generator
+where regulatingequipmenttype = 'GENERATOR'
+  and regulatingequipmentid = generator.id
+  and regulatingpoint.networkuuid = generator.networkuuid
+  and regulatingpoint.variantnum = generator.variantnum);
+-- vsc
+update regulatingpoint
+set regulating = (select vscconverterstation.voltageregulatoron
+from vscconverterstation
+where regulatingequipmenttype = 'VSC_CONVERTER_STATION'
+  and regulatingequipmentid = vscconverterstation.id
+  and regulatingpoint.networkuuid = vscconverterstation.networkuuid
+  and regulatingpoint.variantnum = vscconverterstation.variantnum);
+-- shunt
+update regulatingpoint
+set regulating = (select shuntcompensator.voltageregulatoron
+from shuntcompensator
+where regulatingequipmenttype = 'SHUNT_COMPENSATOR'
+  and regulatingequipmentid = shuntcompensator.id
+  and regulatingpoint.networkuuid = shuntcompensator.networkuuid
+  and regulatingpoint.variantnum = shuntcompensator.variantnum);
```

Comment:
This migration does not work...
I think it should be changed to something like:
```sql
UPDATE regulatingpoint
SET regulating = generator.voltageregulatoron
FROM generator
WHERE regulatingpoint.regulatingequipmenttype = 'GENERATOR'
  AND regulatingpoint.regulatingequipmentid = generator.id
  AND regulatingpoint.networkuuid = generator.networkuuid
  AND regulatingpoint.variantnum = generator.variantnum;
```
otherwise on my db, it sets everything to null except for svc which seems correct

Reply:
true